### PR TITLE
Add "includeCustom" argument for getValidPedModels clientside

### LINKED ISSUE #2551: Add "withCustom" argument for getValidPedModels on client side
**Is your feature request related to a problem? Please describe.**
``getValidPedModels`` on client-side returns also custom ID's from engineRequestModel. If we get valid ped model on client-side and trigger it to server-side where we create ped then we get warning and createPed will returns false because id is custom model.

**Describe the solution you'd like**
The best solution for this will be add a ``withCustom`` argument (default true). This argument should working only on client-side and returns valid ped models including custom models or without custom models.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Fernando-A-Rocha**
Fixes #2551

### CODE REVIEW

File: Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp [hunk 1]
```diff
@@ -12,27 +12,40 @@
 
 int CLuaFunctionDefs::GetValidPedModels(lua_State* luaVM)
 {
-    int iIndex = 0;
+    bool includeCustom = true;
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadBool(includeCustom);
```
Comment (TheNormalnij) [inline Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp]: is `includeCustom` enabled bu default? It looks like `argStream.ReadBool(includeCustom);` rewrites it with `false`.
i guess it should be `argStream.ReadBool(includeCustom, true);`

Comment (Fernando-A-Rocha) [reply Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp]: ah yes it should be enabled by default, let me edit this

File: Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp [hunk 2]
```diff
@@ -12,27 +12,40 @@
 
 int CLuaFunctionDefs::GetValidPedModels(lua_State* luaVM)
 {
-    int iIndex = 0;
+    bool includeCustom = true;
```
Comment (TheNormalnij) [inline Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp]:
```suggestion
    bool includeCustom;
```

**Review by TheNormalnij — DISMISSED** [review]

File: Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp [hunk 3]
```diff
@@ -12,27 +12,40 @@
 
 int CLuaFunctionDefs::GetValidPedModels(lua_State* luaVM)
 {
-    int iIndex = 0;
+    bool includeCustom;
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadBool(includeCustom, true);
+
+    auto modelManager = g_pClientGame->GetManager()->GetModelManager();
```
Comment (TracerDS) [inline Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp]: `auto*`?

**Review by TracerDS — APPROVED** [review]

File: Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp (hunk 3 above)
Comment (Fernando-A-Rocha) [reply Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp]: Why I just copied this from elsewhere

Comment (TracerDS) [reply Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp]: Well, guidelines direct you towards the usage of `auto*`. Though do as you will

Comment (Fernando-A-Rocha) [reply Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp]: My bad I missed that part

**Review by TheNormalnij — APPROVED** [review]

**Comment by Fernando-A-Rocha** [comment]
Sorry for the iterations, I'm new, still getting used to c++ and didn't pay much attention this time 😇

**Review by TracerDS — APPROVED** [review]

**Pull request merged**